src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/homeoffice/HomeOfficeFtpaApplicationDecisionPersonalisation.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.homeoffice;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.ApplicantType;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.ccd.field.YesOrNo;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.EmailNotificationPersonalisation;
+import uk.gov.hmcts.reform.iacasenotificationsapi.infrastructure.PersonalisationProvider;
+import uk.gov.hmcts.reform.iacasenotificationsapi.infrastructure.config.GovNotifyTemplateIdConfiguration;
+
+@Service
+public class HomeOfficeFtpaApplicationDecisionPersonalisation implements EmailNotificationPersonalisation {
+
+    private final GovNotifyTemplateIdConfiguration govNotifyTemplateIdConfiguration;
+    private final PersonalisationProvider personalisationProvider;
+    private final String homeOfficeEmailAddress;
+
+
+    public HomeOfficeFtpaApplicationDecisionPersonalisation(
+        GovNotifyTemplateIdConfiguration govNotifyTemplateIdConfiguration,
+        PersonalisationProvider personalisationProvider,
+        @Value("${allowedAppealHomeOfficeEmailAddress}") String homeOfficeEmailAddress) {
+        this.govNotifyTemplateIdConfiguration = govNotifyTemplateIdConfiguration;
+        this.personalisationProvider = personalisationProvider;
+        this.homeOfficeEmailAddress = homeOfficeEmailAddress;
+    }
+
+    @Override
+    public String getTemplateId(AsylumCase asylumCase) {
+        YesOrNo applicant = this.personalisationProvider.getApplicantType(asylumCase).equals(ApplicantType.RESPONDENT) ? YesOrNo.YES : YesOrNo.NO;
```

Comment:
applicant variable is `YesOrNo.Yes` when variable equals `ApplicantType.RESPONDENT`, why?